# This message describes the current state of the hand as in whether
# it is open or close
bool open